### ISSUE TITLE
Key error in the response 

### ISSUE THREAD

**Issue by ketanbhatt** [indradhanush/birthday-thanks#2]
This https://github.com/indradhanush/birthday-thanks/blob/master/thanks_bot.py#L40 throws:
`KeyError: 'data'`

**Comment by indradhanush** [comment]
Hey, it could be an api change on `facebook's` part. I will look into it, but feel free to submit a patch if you can figure out the issue in the meantime.

**Comment by ketanbhatt** [comment]
Yes looks like an api change as Facebook blocked my account temporarily :D
If I can figure it out, I will. See if you can give some time too, it was my bday yesterday :P
